plugins/rapid7_insightvm/komand_rapid7_insightvm/triggers/scan_completion/trigger.py

```diff
@@ -44,22 +43,14 @@ def run(self, params={}):
             results = self.get_results_from_latest_scan(params=params, scan_id=int(latest_scan_id))
 
             # Submit scan for trigger
-            for item in results:
-                self.send(
-                    {
-                        Output.ASSET_ID: item.get("asset_id"),
-                        Output.IP: item.get("ip_address"),
-                        Output.HOSTNAME: item.get("hostname"),
-                        Output.VULNERABILITY_INFO: item.get("vulnerability_info"),
-                    }
-                )
+            self.send({Output.ASSETS: results[0], Output.VULNERABILITY_INFO: results[1]})
```

Comment:
really minor, could make results name more meaniful now the method is returning two vars? 

```
asset_results, vuln_results = self.get_results_from_latest_scan(params=params, scan_id=int(latest_scan_id))

# Submit scan for trigger
self.send({Output.ASSETS:asset_results, Output.VULNERABILITY_INFO:vuln_results})

```